python/kiss_icp/pipeline.py

```diff
@@ -166,24 +166,22 @@ def _write_gt_poses(self):
         )
 
     def _run_evaluation(self):
-        if not self.has_gt:
-            print("[WARNING] No GT poses available, skipping evaluation")
-            return
-
+        # Run estimation metrics evaluation, only when GT data was provided
+        if self.has_gt:
+            avg_tra, avg_rot = sequence_error(self.gt_poses, self.poses)
+            ate_rot, ate_trans = absolute_trajectory_error(self.gt_poses, self.poses)
+            self.results.append(desc="Average Translation Error", units="%", value=avg_tra)
+            self.results.append(desc="Average Rotational Error", units="deg/m", value=avg_rot)
+            self.results.append(desc="Absolute Trajectory Error (ATE)", units="m", value=ate_trans)
+            self.results.append(desc="Absolute Rotational Error (ARE)", units="rad", value=ate_rot)
+
+        # Run timing metrics evaluation, always
         def _get_fps():
             total_time_s = sum(self.times) * 1e-9
             return float(len(self.times) / total_time_s)
 
-        # Run metrics evaluation
-        avg_tra, avg_rot = sequence_error(self.gt_poses, self.poses)
-        ate_rot, ate_trans = absolute_trajectory_error(self.gt_poses, self.poses)
-        avg_fps = int(np.floor(_get_fps()))
-        avg_ms = 1e3 * (1 / avg_fps)
-
-        self.results.append(desc="Average Translation Error", units="%", value=avg_tra)
-        self.results.append(desc="Average Rotational Error", units="deg/m", value=avg_rot)
-        self.results.append(desc="Absolute Trajectory Error (ATE)", units="m", value=ate_trans)
-        self.results.append(desc="Absolute Rotational Error (ARE)", units="rad", value=ate_rot)
+        avg_fps = int(np.ceil(_get_fps()))
```

Comment:
Taking the opportunity here to always spit the timing results, no matter if GT is available or not